src/Session.cpp

```diff
@@ -1,27 +1,27 @@
 #include "../include/Session.h"
 #include <fstream>
 
-Session::Session(const std::string &configFilePath):
-    content(std::vector<Watchable*>()),
-    actionsLog(std::vector<BaseAction*>()),
-    userMap(std::unordered_map<std::string, User*>()),
-    activeUser(nullptr)
-{
+
+Session::Session(const std::string& configFilePath) :
+        content(std::vector<Watchable*>()),
+        actionsLog(std::vector<BaseAction*>()),
+        userMap(std::unordered_map<std::string, User*>()),
+        activeUser(nullptr) {
     this->fillContentFromJson(configFilePath);
     User* defaultUser = new LengthRecommenderUser("default");
     userMap["default"] = defaultUser;
     activeUser = defaultUser;
 }
 
-Session::~Session()
-{
+Session::~Session() {
     this->clean();
 }
-void Session::split(std::string &actionString, std::vector <std::string>& actionParams, std::string delimiter) {
+
+void
+Session::split(std::string& actionString, std::vector<std::string>& actionParams, std::string delimiter) {
```

Comment:
Seems a little awkward to me

Reply:
You're right
It shouldn't happen
I didn't see it